pages/faq.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import Layout from "../src/components/layouts/layout";
+
+const Faq = () => (
+  <div>
+    <Layout title="FAQ" />
+  </div>
+);
```

Comment:
We could also add a small message here that FAQs are coming soon?

Reply:
Added a Coming Soon message